specification/metrics/semantic_conventions/system-metrics.md

```diff
@@ -33,12 +33,14 @@ instruments not explicitly defined in the specification.
-|                        |                                                                                                          |       |                                                   |            | cpu              | CPU number [0..n-1]                 |
-| system.cpu.utilization | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
-|                        |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+| Name                      | Description                                                                                              | Units | Instrument Type ([*](README.md#instrument-types)) | Value Type | Attribute Key(s) | Attribute Values                    |
+| ------------------------- | -------------------------------------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | ---------------- | ----------------------------------- |
+| system.cpu.time           |                                                                                                          | s     | Counter                                           | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                          |       |                                                   |            | cpu              | CPU number [0..n-1]                 |
+| system.cpu.utilization    | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+| system.cpu.count.physical | Reports the and number of available physical CPUs - may changes over time                                | 1     | UpDownCounters                                    | Int64      | device           | (identifier e.g. `machine-id`)      |
+| system.cpu.count.virtual  | Reports the and number of available virtual CPUs - may changes over time                                 | 1     | UpDownCounters                                    | Int64      | device           | (identifier e.g. `machine-id`)      |
```

Comment:
`machine-id` sounds like a resource attribute to me (it is the value of `host.id` for non-containerized, non-cloud systems after all https://github.com/open-telemetry/semantic-conventions/blob/main/specification/resource/semantic_conventions/host.md) . I think we should leave the `Attribute values` list empty for these two.

Since we do not specify the resource attributes added to these metrics here, we can discuss whether we should add `host.id` to the metrics generated by the hostmetrics receiver on the Collector repository.

cc @dmitryax any thoughts on this?

Reply:
I'm not sure what `device` is and why it's needed as an attribute for this metric while other CPU metrics don't have it. Sound redundant to me. And `host.id` is a resource attribute defined in the conventions and added by resourcedetection processor.

Reply:
Alright, I think we are in agreement, since the `device` attribute was removed we can mark this one as resolved cc @frzifus

Reply:
oh yes, but it seems I made a mistake in the last commit https://github.com/open-telemetry/semantic-conventions/pull/99/commits/d01a81ae2f62d1da3a643b577fe99945dbfe8feb. Let me fix that.

Reply:
fixed it.